package/python-kiwi-spec-template

```diff
@@ -80,6 +80,8 @@ and cloud systems like Xen, KVM, VMware, EC2 and more.
 %package -n python%{python3_pkgversion}-kiwi
 Summary:        KIWI - Appliance Builder Next Generation
 Group:          Development/Languages/Python
+Obsoletes:      python2-kiwi
```

Comment:
IIRC with @Conan-Kudo we talked about including `Conflicts: python2-kiwi` together with the `Obsoletes` a while ago, as result of that it this approach was suggested in https://bugzilla.suse.com/show_bug.cgi?id=1156677 

Also would be nice to mention bsc#1156677 in the commit message.

Reply:
ok done